modules/release-notes/pages/index.adoc

```diff
@@ -67,6 +67,47 @@ TigerGraph Kubernetes Operator allows you to automate operations such as the cre
 == Known Issues
 
 
+|===
+|In version |Description |Status
+
+|3.6.0
+|If a graph on an instance contains a loading job that uses `gsql_split_by_space()` token function, exporting the graph might lead to GPE dysfunction.
+|Unresolved
+
+|3.6.0
+|Accumulating a `DATETIME` object returned from the `to_datetime()` function to a BagAccum inside an ArrayAccum causes GPE dysfunction.
+|Unresolved
+
+|3.6.0
+|Calling `vertex.edgeAttribute()` function with `STRING COMPRESS` data type causes GPE crash
+|Unresolved
+
+|3.6.0
+|Interpreting a query that uses bitwise accumulators causes GPE dysfunction.
+|Unresolved
+
+|3.6.0
+|When vertices are automatically created during edge loading, the primary key of the of vertices created are not written to vertex attributes, even if the vertex definition has `primary_key_as_attribute` set to true.
```

Comment:
Let's remove "even if the vertex definition has `primary_key_as_attribute` set to true."